app/models/book.rb

```diff
@@ -0,0 +1,6 @@
+class Book < ApplicationRecord
+  has_many :copies
+  validates :title, presence: true
+  validates :author, presence: true
+  validates :isbn, presence: true
```

Comment:
Another validation could be to ensure that the isbn is numerical https://guides.rubyonrails.org/active_record_validations.html#numericality